docs/releases/2.2.txt

```diff
@@ -363,6 +363,12 @@ Use this instead::
 
         alias = property(operator.attrgetter('base'))
 
+Cache
+-----
+
+:ref:`Database Cache backend <database-caching>` will raise ``TypeError``
+when get/delete method receives an integer key.
```

Comment:
It is a small change that can go to the `Miscellaneous` section. Maybe 
```
* Providing an integer in the ``key`` argument of the :meth:`.cache.delete` or
  :meth:`.cache.get` now raises :exc:`ValueError`.
```

Reply:
Thanks for the suggestion.
I moved it to `Miscellaneous` section of `Backwards incompatible changes in 2.2` and replaced your suggested text.